src/binding/defaultBindings/css.js

```diff
@@ -8,7 +8,7 @@ ko.bindingHandlers['css'] = {
                 ko.utils.toggleDomNodeCssClass(element, className, shouldHaveClass);
             });
         } else {
-            value = String(value || ''); // Make sure we don't try to store or set a non-string value
+            value = String(ko.utils.stringTrim(value) || ''); // Make sure we don't try to store or set a non-string value
```

Comment:
Not a big issue, but I think the `ko.utils.stringTrim` should probably be outside the `String(value || '')` i.e. `ko.utils.stringTrim(String(value) || '')`. The `stringTrim` function [looks pretty safe](https://github.com/knockout/knockout/blob/8535832f0bd22aeecb2aba53311bf55b77a7d6ad/src/utils.js#L294) though.

Worth noting: http://stackoverflow.com/a/3945236/19212 ; we could probably get rid of the `String` constructor itself, since `stringTrim` calls `toString`, meaning we could just use `ko.utils.stringTrim(value || '')`.